packages/react-components/react-table/src/components/Table/useTableStyles.styles.ts

```diff
@@ -20,6 +20,7 @@ const useTableLayoutStyles = makeStyles({
 const useFlexLayoutStyles = makeStyles({
   root: {
     display: 'block',
+    minWidth: 'fit-content',
```

Comment:
might be a good idea to add docs here as well ( which is part of PR description ) . do we have this covered by browser tests ?